playbooks/insights_setup.yml

```diff
@@ -1,7 +1,13 @@
 ---
-# Placeholder playbook for setting up Insights services (advisor, drift, patch, vulnerability, and policies)
-- hosts: localhost
+# Steps required to get your system ready to use the Insights Client:
+# 1. Yum install the insights-client
+# 2. Register the insights-client
+# 3. Modify file permissions
+- hosts: all
+  become: yes
   tasks:
-    - name: Setup Insights
-      debug:
-        msg: "Insights setup"
+    - name: Install 'insights-client'
```

Comment:
This would perhaps be a good place to sneak in the `yum update rhc` workaround for https://issues.redhat.com/browse/RHCLOUD-13282

@subpop @lphiri FYI

Reply:
Seems fair to me. We've always planned to have cloud-controlled updates of rhc, so I wouldn't necessarily consider this to be sneaky. :)

Reply:
I can add a yum update rhc as the first step.

Reply:
Added two additional tasks. One to update rhc to latest and another for rhc-worker-playbook. @lphiri Should we also be updating the insight-client to latest?

Reply:
@Jason-RH - I don't see any issues with that, as long its documented behavior.

Reply:
:+1: If we aren't removing the client when we disable it we should probably update.